genes/run.rb

```diff
@@ -0,0 +1,15 @@
+require './horse_dictionary'
+
+horse_dictionary = HorseDictionary.new
+
+father = horse_dictionary.get('Amber')
+mother = horse_dictionary.get('Amber')
+
+a_feature, t_feature = father.combine(mother)
+
+a_feature.each do |a_gene|
+  t_feature.each do |t_gene|
+    puts "#{horse_dictionary.get_breed_name(HorseBreed.new(a_gene.allele_type, t_gene.allele_type))[0]} " \
+          "#{((a_gene.probability * t_gene.probability) / 10_000.0)}"
```

Comment:
You could represent probabilities with [`Rational`](https://apidock.com/ruby/Rational).
This way you'd not need to divide by `10_000`.
Then you could also display probabilities as fractions:
```ruby
a_feature.each do |a_gene|
  t_feature.each do |t_gene|
    child_probability = a_gene.probability * t_gene.probability
    puts "#{horse_dictionary.get_breed_name(HorseBreed.new(a_gene.allele_type, t_gene.allele_type))[0]} " \
      "#{child_probability} (#{child_probability.to_f})"
  end
end

# Amber 1/4 (0.25)
# Begonia 1/8 (0.125)
# Amaranth 1/8 (0.125)
# Alloy orange 1/8 (0.125)
# Almond 1/16 (0.0625)
# Alizarin crimson 1/16 (0.0625)
# Blush 1/8 (0.125)
# Cadmium 1/16 (0.0625)
# Bittersweet 1/16 (0.0625)
```